### PR TITLE
management sdk: signalr client tool

### PR DESCRIPTION
**Pull request by realwanpengli**
signalr clients listen to messages

### CODE REVIEW

File: samples/Management/SignalRClient/Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            {
+                var negotiateEndpoint = negotiateEndpointOption.Value() ?? DefaultNegotiateEndpoint;
+                var userIds = userIdOption.Values;
+
+                var connections = (from userId in userIds
+                                   select new
+                                   {
+                                       Connection = new HubConnectionBuilder().WithUrl(negotiateEndpoint.TrimEnd('/') + $"?user={userId}").Build(),
+                                       UserId = userId
+                                   }).ToList();
```
Comment (vicancy) [inline samples/Management/SignalRClient/Program.cs]: no need to `ToList`

File: samples/Management/SignalRClient/Program.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+                foreach (var conn in connections)
+                {
+                    conn.Connection.On(Target, (string message) =>
```
Comment (vicancy) [inline samples/Management/SignalRClient/Program.cs]: move into where connection is created?

File: samples/Management/SignalRClient/Program.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        private const string DefaultNegotiateEndpoint = "http://localhost:5000";
+        private const string Target = "Target";
+
+        static void Main(string[] args)
+        {
+            var app = new CommandLineApplication();
+            app.FullName = "Azure SignalR Management Sample";
+            app.HelpOption("--help");
+
+            var negotiateEndpointOption = app.Option("-n|--negotiate", $"Set negotiation endpoint. Default value: {DefaultNegotiateEndpoint}", CommandOptionType.SingleValue, true);
```
Comment (chenkennt) [inline samples/Management/SignalRClient/Program.cs]: For customer it's not a negotiate endpoint, just a hub endpoint

Comment (realwanpengli) [reply samples/Management/SignalRClient/Program.cs]: right

File: samples/Management/SignalRClient/Program.cs [hunk 4]
```diff
@@ -30,26 +30,32 @@ static void Main(string[] args)
                 var hubEndpoint = hubEndpointOption.Value() ?? DefaultHubEndpoint;
                 var userIds = userIdOption.Values != null && userIdOption.Values.Count() > 0 ? userIdOption.Values : new List<string>() { "User" };
```
Comment (vicancy) [inline samples/Management/SignalRClient/Program.cs]: `Count`

**Pull request merged**